libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -15,4 +15,13 @@ export class ReadingListComponent {
   removeFromReadingList(item) {
     this.store.dispatch(removeFromReadingList({ item }));
   }
+
+  toggleFinishReadingList(item: ReadingListItem) {
+    const finishedBook = {
```

Comment:
Once you remove toggle finish book part, you need to update this section also.

Reply:
Done